.github/workflows/e2e-test.yml

```diff
@@ -125,9 +128,12 @@ jobs:
       - name: Run E2E Tests
         run: npm run test:e2e
 
-      # Upload screenshots if any
-      - name: Upload screenshots
+      # Archive screenshots if any
+      - name: Archive e2e test screenshots
+        if: ${{ always() }}
         uses: actions/upload-artifact@v2
         with:
-          name: screenshots
-          path: screenshots
+            name: e2e-screenshots
+            path: tests/e2e/screenshots
```

Comment:
Was there a change to where the screenshots get saved? It happens [here](https://github.com/Automattic/woocommerce-payments/blob/63497bd134466478aeb34770b725e52a33cdef39/tests/e2e/utils/helpers.js#L18) but maybe when running locally it depends on where the tests are run from? Not seeing the artifact on recent runs (e.g. [here](https://github.com/Automattic/woocommerce-payments/actions/runs/1755888388)).

Reply:
@dechov We moved to use the `useE2EJestConfig` provided by `e2e-environment` which sends the failed screenshot messages directly to `#wcpay-bots` channel in slack, which might have overridden the old artifact behavior to only create artifacts for failure cases, see [this run](https://github.com/Automattic/woocommerce-payments/actions/runs/1789277846) for reference.
Do we need the screenshots for success scenarios as well?

Reply:
Ah, I see 👍

Not that we need the regular ones but there is some theoretical benefit to having snapshots from each distinct screen from the normal run, to ensure that there aren't visual regressions that wouldn't be caught by specific assertions. What do you think of having a "failure screenshots" artifact in addition to the earlier one?

Reply:
I feel unless there is an alert/checking process in place, we wouldn't really monitor the successful ones regularly, and manually checking visual regressions could be tricky across runs.
We could either ask GS to do some sanity or have explicit visual regression tests in place for the same but that's just my opinion, we could discuss it internally to see what other folks feel about the same.